VideoLocker/src/main/java/org/edx/mobile/loader/CoursesAsyncLoader.java

```diff
@@ -21,22 +21,26 @@
     private Observer mObserver;
 
     IEdxEnvironment environment;
+    UserAPI api;
```

Comment:
Maybe you forgot to initialize the injections, i.e. call `RoboGuice.injectMembers()`?